README.md

```diff
@@ -1,5 +1,5 @@
 ## GraphStorm
-|[Document and Tutorial Site](https://graphstorm.readthedocs.io/en/latest/) |
+| [Document and Tutorial Site](https://graphstorm.readthedocs.io/en/latest/) | [GraphStorm Paper](https://arxiv.org/abs/2406.06022) |
```

Comment:
Should we add a citation section in the repo homepage? For example,

Citation
@article{Da2024GraphStormKDD,
      title={GraphStorm: all-in-one graph machine learning framework for industry applications}, 
      author={Da Zheng, Xiang Song, Qi Zhu, Jian Zhang, Theodore Vasiloudis, Runjie Ma, Houyu Zhang, Zichen Wang, Soji Adeshina, Israt Nisa, Alejandro Mottini, Qingjun Cui, Huzefa Rangwala, Belinda Zeng, Christos Faloutsos, and George Karypis},
      year={2024},
      eprint={2406.06022},
      archivePrefix={arXiv}
}